galsim/wfirst/wfirst_psfs.py

```diff
@@ -0,0 +1,460 @@
+        # index that is 9 lower, except for certain sign flips (astig1, coma2, trefoil2) that result
+        # in symmetry about the FPA y axis (except for the struts).
+        read_SCA = SCA
+        if SCA >= 10:
+            if aberration_list[SCA-9] is not None:
+                read_SCA = -1
+                tmp_aberrations = aberration_list[SCA-9]
+                tmp_aberrations *= np.array([1.,1.,1.,1.,1.,-1.,1.,1.,-1.,1.,-1.,1.])
+                aberration_list.append(tmp_aberrations)
+            else:
+                read_SCA -= 9
+
+        # If we got here, then it means we have to read in the aberrations.
+        if read_SCA > 0:
+            aberrations = _read_aberrations(read_SCA)
+            if read_SCA != SCA:
+                aberrations *= np.array([1.,1.,1.,1.,1.,-1.,1.,1.,-1.,1.,-1.,1.])
+            aberration_list.append(aberrations)
+
+        use_aberrations = aberration_list[SCA]
+        if extra_aberrations is not None:
+            use_aberrations += extra_aberrations
+
+        # Now set up the PSF for this SCA, including the option to simplify the pupil plane.
+        if verbose: print '   ... SCA',SCA
+        if not achromatic:
+            if approximate_struts:
+                PSF = galsim.ChromaticOpticalPSF(
+                    lam=zemax_wavelength,
+                    diam=galsim.wfirst.diameter, aberrations=use_aberrations,
+                    obscuration=galsim.wfirst.obscuration, nstruts=6)
+            else:
+                PSF = galsim.ChromaticOpticalPSF(
+                    lam=zemax_wavelength,
+                    diam=galsim.wfirst.diameter, aberrations=use_aberrations,
+                    obscuration=galsim.wfirst.obscuration,
+                    pupil_plane_im=galsim.wfirst.pupil_plane_file,
+                    oversampling=1.2, pad_factor=2.)
+            if n_waves is not None:
+                PSF.setupInterpolation(waves=np.linspace(blue_limit, red_limit, n_waves),
+                                       oversample_fac=1.5)
+        else:
+            tmp_aberrations = use_aberrations * zemax_wavelength / achromatic_lam
+            if approximate_struts:
+                PSF = galsim.OpticalPSF(lam=achromatic_lam, diam=galsim.wfirst.diameter,
+                                        aberrations=tmp_aberrations,
+                                        obscuration=galsim.wfirst.obscuration, nstruts=6)
+            else:
+                PSF = galsim.OpticalPSF(lam=achromatic_lam, diam=galsim.wfirst.diameter,
+                                        aberrations=tmp_aberrations,
+                                        obscuration=galsim.wfirst.obscuration,
+                                        pupil_plane_im=galsim.wfirst.pupil_plane_file,
+                                        oversampling=1.2, pad_factor=2.)
+
+        PSF_list.append(PSF)
+
+    return PSF_list
+
+def tabulatePSFImages(PSF_list, image_filename, data_filename, bandpass_list=None,
```

Comment:
You have `tabulate` and `getStored`.  Recommend using the same verb for both.  I like "store" better.  In fact, I like "load" even better than "getStored".

In fact, to be even more parallel, you should omit the "Images" part of this name.  So I'd suggest `storePSFs` and `loadPSFs` for the two functions.